content/docs/security.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Security
+description: Securing TiKV 
+weight: 7
+draft: false
+---
+
+This page discusses how to secure your TiKV deployment. Learn how to:
+
+* Use [Transport Layer Security](#transport-layer-security-tls) to encrypt connections between TiKV nodes
+* Use [On-Disk Encryption](#on-disk-encryption) to encrypt the data that TiKV reads and writes to disk
+* [Report vulnerabilities](#reporting-vulnerabilities)
+
+## Transport Layer Security (TLS)
+
+It is possible to use TLS encryption with TiKV. To start, you need to prepare the certificates for the deployment.

```

Comment:
```suggestion
To use TLS encryption with TiKV, you need to prepare and configure the certificates for the deployment.
```

Reply:
Could you say something about why a user might want to use TLS with TiKV (and same with ODE below)? And the benefits it brings for security